src/progress-steps/__tests__/progress-steps-rtl.scenario.js

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+
+import {Block} from '../../block/index.js';
+import {Button} from '../../button/index.js';
+import {ThemeProvider, LightTheme} from '../../index.js';
+import {ProgressSteps, Step} from '../index.js';
+
+export const name = 'progress-steps-rtl';
+
+export const component = () => <ProgressStepsContainer />;
+
+class ProgressStepsContainer extends React.Component<{}, {current: number}> {
+  state = {current: 0};
+  render() {
+    return (
+      <ThemeProvider theme={{...LightTheme, direction: 'rtl'}}>
+        <ProgressSteps current={this.state.current}>
```

Comment:
can we require the original story, and export the wrapped version from here, so we don't duplicate the code?